commit 09ab686d8009503b8a9ccc8947a617d80c038bc8 (refs/changes/80/280/2)
Author: Julien Gilli <julien.gilli@joyent.com>
Date:   2016-08-12T16:14:50+00:00 (3 years, 2 months ago)
    
    TOOLS-1519 node-sdc-client's make ufds_test target cannot run UFDS tests suite
    
    Reviewed by: Pedro P. Candel <pedro@joyent.com>

diff --git a/test/ufds.tests.js b/test/ufds.test.js
similarity index 100%
rename from test/ufds.tests.js
rename to test/ufds.test.js
